packages/core/src/claim/Claim.ts

```diff
@@ -289,7 +289,8 @@ export function fromCTypeAndClaimContents(
   claimContents: IClaim['contents'],
   claimOwner: DidUri
 ): IClaim {
-  if (ctypeInput.schema) {
+  // TODO: is schema really optional? Yes = fix type. No = use isICType here.
+  if ('schema' in ctypeInput) {
```

Comment:
Need a decision here

Reply:
The type definition of `ICType` does not define schema to be optional and that is good and correct.
Thinking about JS people though, it is a good idea to use our typeguards to protect against mistakes though.

Reply:
Added the type guard in https://github.com/KILTprotocol/sdk-js/pull/612/commits/e2abc8dc05b0343ea70e1959aa2a21578d445271